lisa/transformers/scripts_execution_transformer.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+from lisa.util import LisaException
+
+
+class ScriptInterpreter(str, Enum):
+    BASH = "bash"
+
+
+@dataclass_json()
+@dataclass
+class ScriptEntry:
+    script: str = ""
+    interpreter: ScriptInterpreter = ScriptInterpreter.BASH
+    args: Optional[str] = None
+
```

Comment:
Add one more field `expected_exit_code`. The default value is `0`. It can be set to `None`, and ignore the exit code.

Reply:
updated